x-pack/plugins/cloud_security_posture/public/pages/rules/rules_container.tsx

```diff
@@ -93,6 +94,43 @@ export const RulesContainer = () => {
     params.benchmarkVersion
   );
 
+  const rulesStates = useCspGetRulesStates();
+  const arrayRulesStates = Object.values(rulesStates.data || {});
+  const filteredRulesStates: any[] = arrayRulesStates.filter(
+    (e: any) =>
+      e.benchmark_id === params.benchmarkId && e.benchmark_version === 'v' + params.benchmarkVersion
+  );
+
+  const allNewRulesItems: CspBenchmarkRulesWithStatus[] = useMemo(() => {
```

Comment:
let's rename to `rulesWithStatus`
1. will fit with the type
2. will be better in the future when we add more statuses